fastlane/Fastfile

```diff
@@ -114,6 +114,15 @@ lane :run_e2e_test do |options|
   UI.user_error!('Tests have failed!') if result.include?('Failures')
 end
 
+lane :lint_e2e_test do |options|
+  Dir.chdir('..') do
+    ['spotlessApply', 'apiDump', 'detekt'].each do |task|
+      sh("./gradlew :stream-chat-android-e2e-test:#{task}")
+      sh("./gradlew :stream-chat-android-compose-sample:#{task}") unless task == 'apiDump'
+    end
+  end
+end
```

Comment:
I'm wondering what that is used for.

`spotlessApply` and `detekt` are already called as part of the PR workflow.

Regarding `apiDump`, I would say that it should be kept as is, as a pre-push hook only.

Reply:
It's not used anywhere. Tbh I just added it for my local usage because it was quite painful to get the PR blocked because of lint issues and then trying to recall all these commands 🙂

Reply:
In that case, shouldn't `compose-sample` and `apiDump` be removed from there?

tip: a single call with both `spotlessApply` and `detekt` is more performant than two calls, for example:

`./gradlew :stream-chat-android-e2e-test:spotlessApply :stream-chat-android-e2e-test:detekt`

Reply:
Yeah, as Jc opened a PR regarding `apiDump` I already removed it from this PR.

The tests are part of `compose-sample` so I can't remove it.

`./gradlew :stream-chat-android-e2e-test:spotlessApply :stream-chat-android-e2e-test:detekt` updated 👍